static/js/util/ProfileTab.js

```diff
@@ -58,14 +60,44 @@ class ProfileTab extends React.Component {
       {value: MASTERS, label: "Master's or professional degree"},
       {value: DOCTORATE, label: "Doctorate"}
     ];
+    this.prevButtonText = "Previous";
+    this.nextButtonText = "Save And Continue";
   }
 
+  stepBack = () => {
+    this.context.router.push(this.prevUrl);
+  };
+
   saveAndContinue = () => {
-    let lastTab = this.nextUrl === "/dashboard";
-    saveAndContinue.call(this, this.constructor.prototype.validation, lastTab).then(() => {
+    saveProfileStep.call(this, this.constructor.prototype.validation, !!this.isLastTab).then(() => {
       this.context.router.push(this.nextUrl);
     });
-  }
+  };
+
+  progressControls = (cellWidth=10) => {
```

Comment:
i think this component is defined in the right place. it is relevant to all components that subclass from `ProfileTab`. profile_edit.js may have most of our current low-level components, but i don't think that's a reason to put _all_ low-level components there. a lot of them make sense in there  because they can be used across many different components (`boundTextField`, `boundStateSelectField`, etc.). this `progressControls` component is only intended to be used by `ProfileTab` children